### ISSUE TITLE
NA Birds 4

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#4]
Dataset: NA Birds
Data: 10%
Model: Best iNat21 with hierarchy

Use a pre-trained swin-v2-base model pre-trained on iNat21 with hierarchy with the best validation loss, fine-tune it on NA Birds with 10% of training data, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
